src/components/search/SearchResults.jsx

```diff
@@ -113,19 +111,20 @@ const SearchResults = ({
 };
 
 SearchResults.propTypes = {
-  searchResults: PropTypes.shape({
-    nbHits: PropTypes.number,
-  }),
   searchState: PropTypes.shape({
     query: PropTypes.string,
     page: PropTypes.number,
   }).isRequired,
-  isSearchStalled: PropTypes.bool.isRequired,
```

Comment:
[Nit] What convention are you rearranging these by? I can kinda see isRequired listed at the top and the nonRequired props at the bottom but they don't seem to be in any order beyond that?

Reply:
Yeah, exactly. I try to keep required props at the top but non-required props after that. Otherwise, order doesn't really matter and haven't really found it worthwhile to try and keep them alphabetical or anything like that.

Reply:
I super like things being in alphabetical order but it's not a standard I try to force on other people lol. Just wanted to know if there was a secondary convention sorting convention that I was missing like what proptype they are or "alphabetical but the word 'is' doesn't count like the word 'the' sometimes doesn't count and then error just on the bottom"